data/language/dutch.txt

```diff
@@ -53,9 +53,9 @@ STR_0051    :Attractie
 STR_0052    :Attractie
 STR_0053    :Hyper-Twisterachtbaan
 STR_0054    :Houten achtbaan
-STR_0055    :Zijfrictieachtbaan
+STR_0055    :Zijfrictie-achtbaan
 STR_0056    :Wilde muis
-STR_0057    :Multidimensieachtbaan
+STR_0057    :Multidimensie-achtbaan
```

Comment:
Ik heb het inderdaad hierom gedaan. Beide varianten zijn inderdaad correct, maar ik denk toch dat we het beste voor deze versie kunnen gaan wegens de optimale leesbaarheid.